Vagrantfile

```diff
@@ -15,6 +15,4 @@ Vagrant.configure("2") do |config|
 
   config.vm.provision :shell, path: "vagrant-provision.sh"
 
-  config.vm.synced_folder ".", "/vagrant"
```

Comment:
We still need some kind of synched folder, and on the `libvirt` backend, this was the only way. If it's only useful for `libvirt` then we can hide it behind an `if`.

Reply:
Eh never mind, @gabriel-v says it's usually not reliable.